packages/components/src/popover/stories/index.tsx

```diff
@@ -49,6 +49,7 @@ const meta: ComponentMeta< typeof Popover > = {
 		onClose: { action: 'onClose' },
 		onFocusOutside: { action: 'onFocusOutside' },
 		__unstableSlotName: { control: { type: null } },
+		variant: { options: [ undefined, 'toolbar', 'unstyled' ] },
```

Comment:
```suggestion

```

I assume you added this so it can be set back to `undefined`, but we've been unifying our controls to _not_ do this, and instead use the "Reset controls" button in Storybook itself. It's definitely a trade-off (between usability and keeping these enums up-to-date manually), but at the moment we're trying to leverage the TypeScript data as much as we can.

Reply:
Thanks for the info, wasn't aware of that!